docs/development/releasing.rst

```diff
@@ -227,34 +213,6 @@ and commit this and the ``.mailmap`` changes::
 Open a pull request to merge this into ``main`` and mark it as requiring backporting to
 the release branch.
 
-
-.. _release-procedure-update-ci:
-
-Update continuous integration configuration
--------------------------------------------
-
-Update the continuous integration configuration in the release branch
-to run on all commits rather than use cron jobs. For example, for GitHub actions,
-you should edit the ``ci_cron*.yml`` files and replace the existing ``on`` section
-with e.g.::
-
-   on:
-   push:
-      branches:
-      - v5.0.x
-   pull_request:
-      branches:
-      - v5.0.x
-
-(with the branch name replaced by the appropriate one), and remove any lines that
-look like e.g.::
-
-        if: (github.repository == 'astropy/astropy' && (github.event_name == 'schedule' ...
-
-This is important because once you are on a release branch, it is necessary to make sure
-we are much more careful about not introducing regressions and we cannot always wait for the
-cron jobs to run to carry out the release.
-
```

Comment:
Why is this block deleted ? (It doesn't seem related to this PR)

Reply:
It is no longer needed but I forgot to remove it in the PR that updated the CI config (https://github.com/astropy/astropy/pull/12832) - just thought I'd remove it here since I was editing the release docs anyway.